stories/components/data-table/docs/examples/limited-mode.tsx

```diff
@@ -0,0 +1,141 @@
+import type { DataTableData } from '@semcore/data-table';
+import { DataTable } from '@semcore/data-table';
+import { Text } from '@semcore/typography';
+import { Flex } from '@semcore/ui/base-components';
+import Button from '@semcore/ui/button';
+import React from 'react';
+
+export type LimitedModeExampleProps = {
+  rowsLimit?: number;
+  columnsLimit?: number;
+};
+
+const Demo = (props: LimitedModeExampleProps) => {
+  const { rowsLimit, columnsLimit } = props;
+
+  return (
+    <DataTable
+      data={data}
+      aria-label='Limited table example'
+      defaultGridTemplateColumnWidth='auto'
+      wMax='800px'
+      limit={{
+        fromRow: rowsLimit,
+        fromColumn: columnsLimit,
+        renderOverlay() {
+          return (
+            <Flex alignItems='center' direction='column' gap={3} py={6} wMax={320}>
```

Comment:
Is it possible to add `text-wrap: balance` for this block? This will give the text a more balanced wrapping behavior.

Like this
<img width="597" height="379" alt="image" src="https://github.com/user-attachments/assets/45201c2b-9063-4952-ac2a-d845a0182eb9" />

Instead of this
<img width="592" height="382" alt="image" src="https://github.com/user-attachments/assets/7da37f52-68d4-4049-a4f5-3367b2c6b364" />